docs/schema/conditions-element-workflowinfo-elementaction4.md

```diff
@@ -173,31 +174,31 @@ If the schema defines specific requirements, such as **sequence**, **minOccurs**
 <td align="left"><p>optional</p></td>
 <td align="left"><p>The text that is defined in this attribute is displayed in the rule designer sentence when a conditional branch activity is added to the workflow. The value is not case-sensitive.</p>
 <p>The default value is **Else if** (applies only to the English language version of SharePoint).</p>
-<p>**Example**: `<Conditions Else="Else if">`</p></td>
+<p>**Example**: `&lt;Conditions Else="Else if">`</p></td>
```

Comment:
Not a huge deal, but the repo owner for this repo is preferring an alternative fix for these types of issues - so a better fix may be "<Conditions Else="Else if">" and so on throughout this file.